internal/statestore/testing/consts.go

```diff
@@ -25,8 +25,8 @@ const (
 	PoolIdleTimeout = 10 * time.Second
 	// PoolHealthCheckTimeout is the read/write timeout of a healthcheck HTTP request
 	PoolHealthCheckTimeout = 100 * time.Millisecond
-	// IgnoreListTTL is the time to live duration of Open Match ignore list settings
-	IgnoreListTTL = 500 * time.Millisecond
+	// pendingReleaseTimeout is the time to live duration of Open Match ignore list settings
+	pendingReleaseTimeout = 500 * time.Millisecond
```

Comment:
Would it be 200?

Reply:
This is for a different set of tests.  They don't need to match.